interface/patient_file/encounter/new_form.php

```diff
@@ -4,33 +4,50 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
-include_once("../../globals.php");
+require_once("../../globals.php");
 require_once $GLOBALS['srcdir'].'/ESign/Api.php';
 
+use OpenEMR\Encounter\Services\ViewHelper;
+
 $esignApi = new Esign\Api();
+
+$assets_dir = $GLOBALS['assets_static_relative'];
 ?>
 <?php if (empty($hide)) { // if not included by forms.php ?>
 <html>
 <head>
 <?php } ?>
 <?php html_header_show();?>
 <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $assets_dir;?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
+<link rel="stylesheet" href="<?php echo $assets_dir;?>/font-awesome-4-6-3/css/font-awesome.css" type="text/css">
 
+<script type="text/javascript" src="<?php echo $assets_dir;?>/jquery-min-3-1-1/index.js"></script>
+<script type="text/javascript" src="<?php echo $assets_dir;?>/bootstrap-3-3-4/dist/js/bootstrap.js"></script>
```

Comment:
delete above 5 lines of code (including removing the $css_header include) and just do this here instead(brings in all above stuff and also sets up for rtl):
``````
<?php require($GLOBALS['srcdir'] . '/templates/standard_header_template.php'); ?>
``````